Cloud Application Manager/Troubleshooting/camdce-troubleshooting-tips.md

```diff
@@ -0,0 +1,289 @@
+
+**Solution 1**
+
+Make sure the communication from CAM Agent to CAM appliance works correctly.
+
+1. Connect to the instance by SSH or RDP.
+
+2. Run a ping command to check basic connectivity:
+
+```
+ping -c3 10.0.0.1
+```
+
+**Note:** Replace 10.0.0.1 with your appliance IP or hostname.
+
+If the above command does not get a reply, please take the corrective actions inside the VM, routers and firewalls to allow unrestricted communication between agent and CAM appliance.
+
+If the connectivity seems to be working as expected and the problem persists, connect to the instance over SSH or RDP and grab a copy of the agent `/var/log/elasticbox/elasticbox-agent.log` in the case of Linux or `ProgramData\ElasticBox\elasticbox-agent.log` in the case of Windows (please note that the folder is hidden in Windows and the exact path must be entered to navigate to the folder). Once the log file is collected please attach it when [Contacting support](mailto:incident@CenturyLink.com)
+
+**Solution 2**
+
+Check agent proxy settings (if configured to use one) at `/usr/elasticbox/elasticbox.conf` in linux or `ProgramFiles\ElasticBox\Agent\elasticbox.conf` in windows. If proxy is used for agent, make sure its working correctly and provide stable connectivity to CAM appliance.
+
+**Solution 3**
+
+Install the agent on the instance to bring it online. Then re-run the lifecycle operation.
+
+1. Connect to the instance by SSH or RDP.
+
+2. Install the agent. The command uses the token of the older agent to connect to the instance.
+
+    Linux instances deployed from the Cloud Application Manager Appliance:
+
+    ```
+    curl -ks 10.0.0.1 | sudo bash
+    ```
+
+    **Note:** Replace 10.0.0.1 with your appliance IP or hostname.
+
+    Windows instances deployed from the Cloud Application Manager Appliance (run the command as a PowerShell administrator):
+
+    ```
+    (New-Object Net.WebClient).DownloadString("http://10.0.0.1") | iex
+    ```
+
+If the problem persists connect to the instance over SSH or RDP and grab a copy of the agent `/var/log/elasticbox/elasticbox-agent.log `in the case of Linux or `ProgramData\ElasticBox\elasticbox-agent.log` in the case of Windows (please note that the folder is hidden in Windows and the exact path must be entered to navigate to the folder). Once the log file is collected, please attach it when [Contacting support](mailto:incident@CenturyLink.com)
+
+### Instance is Stuck Deploying
+
+**Cause 1**
+
+An instance can’t update its state because the Cloud Application Manager agent can’t connect to Cloud Application Manager. Poor agent network connectivity can cause this issue.
```

Comment:
An instance can't update its state because the Cloud Application Manager agent can't connect to the Cloud Application Manager system, SaaS or Appliance.  (Some language to make it clear that the agent cant communicate with the backend system, SaaS or Appliance.)